create-plugin-update/action.yml

```diff
@@ -0,0 +1,93 @@
+name: "Grafana Create Plugin Update"
+description: "Runs create-plugin update within a Grafana plugin"
```

Comment:
Could you change this to specify this actually creates a PR? 

`Runs create-plugin update within a Grafana plugin and opens a PR with the changes`

Reply:
Have updated as suggested. 👍